.github/workflows/all.yml

```
@@ -305,3 +305,25 @@ jobs:
         run: |
           mv tools/semgrep/.semgrepignore . & \
           semgrep --verbose --config="r2c" --config="tools/semgrep/rules/example.yaml"
+
+  # Check notice file for update
+  notice_update:
+    name: All - notice_update
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/build-trust/ockam-builder@sha256:cecb1860acd571278b2e7f8ecb3ffe405447ee844615134f93ddd11b1f3e2ca8
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - run: cargo deny --all-features  list --config=tools/cargo-deny/deny.toml --layout crate --format json | ./tools/scripts/release/parseCrates.sh > update.txt
```

We are piping data to the notice file here https://github.com/build-trust/ockam/blob/1ca0082676b5cddd431998b080fc2df13cb7791f/tools/scripts/release/parseCrates.sh#L52 we can't also pipe to update.txt as it'll lead to unprecedented results
```suggestion
      - run: cargo deny --all-features  list --config=tools/cargo-deny/deny.toml --layout crate --format json | ./tools/scripts/release/parseCrates.sh
```